### ISSUE TITLE
font(KbdKhmr): 2 symbols show up as tofu

### ISSUE THREAD

**Issue by Nnyny** [keymanapp/keyboards#3229]
I'm working on [Khmer Angkor version 2.0](https://github.com/keymanapp/keyboards/pull/3228) and notice that the KbdKhmr font display ฿ and ‱ as tofu. Also, hint displays on iOS is slightly different than Android. 

iOS:
![Image](https://github.com/user-attachments/assets/f730b078-d6a6-45a2-a71f-dc8af2070782)

Android:
 ![Image](https://github.com/user-attachments/assets/6be1cdf1-d7dd-4963-847a-93c983999478)

**Comment by Nnyny** [comment]
FYI @mcdurdin .

**Comment by mcdurdin** [comment]
Which hints are the issue? ាំ displays incorrectly on iOS; are there others?

**Comment by mcdurdin** [comment]
FYI @Nnyny, I think this is a compiler bug, so we'll look at fixing this in the compiler in 18.0, so no need to wait for this in order to deploy the keyboard.

The tofu for ฿ and ‱ requires an update to KbdKhmr font I think.

**Comment by Nnyny** [comment]
Noted! Should I add these 2 symbols into  `release\shared\fonts\kbd\kbdkhmr.json` ?

**Comment by mcdurdin** [comment]
> Noted! Should I add these 2 symbols into `release\shared\fonts\kbd\kbdkhmr.json` ?

The symbols can be added to the JSON file but also need to be added to the KbdKhmr font itself.

**Comment by Nnyny** [comment]
So should I wait for the font to be updated before I deploy the keyboard?

**Comment by mcdurdin** [comment]
Yes, perhaps we need to wait on the font (although both of those characters could just be removed from the keyboard, because in my opinion neither of them are critical to the use of Khmer)

**Comment by Nnyny** [comment]
I will remove them instead.
